templates/ohdsi-webapi/scripts/atlas_db_init.sh

```diff
@@ -0,0 +1,18 @@
+#!bin/bash
+
+apk --update add postgresql-client gettext
+admin_user_password="${OHDSI_ADMIN_PASSWORD}${OHDSI_ADMIN_USERNAME}"
+app_user_password="${OHDSI_APP_PASSWORD}${OHDSI_APP_USERNAME}"
+export admin_md5="'md5$(echo -n $admin_user_password | md5sum | awk '{ print $1 }')'"
+export app_md5="'md5$(echo -n $app_user_password | md5sum | awk '{ print $1 }')'"
+
+atlas_create_roles_users_script=$(envsubst < atlas_create_roles_users.sql)
+atlas_create_schema_script=$(envsubst < atlas_create_schema.sql)
+
+printf 'Creating roles and users'
+echo "$atlas_create_roles_users_script" | psql "$MAIN_CONNECTION_STRING"
```

Comment:
If we add `-e` to psql it will echo the commands send to the DB,
it will be good for tracking in logs

Reply:
done